### ISSUE TITLE
Android App

### ISSUE THREAD

**Issue by CallMeChin** [mgilangjanuar/teledrive#195]
An android version of the app would be nice, with feature like auto backup picture(something like google photos)

**Comment by mgilangjanuar** [comment]
Good idea! Actually, we're on it to make a sync button for Google Drive too. And the native mobile app will come 🚀

**Comment by ZeroDot1** [comment]
Yes, an Android app would be really good. :four_leaf_clover:
